modules/feh/hm.nix

```diff
@@ -0,0 +1,12 @@
+{ pkgs, config, lib, ... }:
+
+{
+  options.stylix.targets.feh.enable =
+    config.lib.stylix.mkEnableTarget
+    "the desktop background using Feh"
+    true;
```

Comment:
Having this always enabled could be an issue, as some desktop environments have their own way to set the wallpaper.

Reply:
Fair enough. I though that the desktop environment to have their own way most of the time do not use xinitrc ? But I may be wrong. I could do something like what was done for the system theme, and select a list of wm for which to auto-enable it.

Reply:
Maybe [wallutils](https://github.com/xyproto/wallutils) can help us - it detects the current window manager and sets the wallpaper accordingly. We could remove the code for specific desktop environments and instead call wallutils no matter what. We'd have to prevent it running twice when NixOS and Home Manager are used together, or at least guarantee that the Home Manager wallpaper takes priority.

Reply:
Anyway I was planning to work on another PR after this one to create a wallpaper module with multiple options to set the wallpaper. For now we can replace true by something like : 
```nix
with config.xsession.windowManager; bspwm.enable 
                                 || herbsflutwm.enable 
                                 || i3.enable 
                                 || spectrwm.enable 
                                 || xmonad.enable
```

And maybe we can force it to false if it is enabled on the system config ? But I'm pretty sure if both are enabled it wouldn't fail, the user one should take over (which we should expect anyway if the images are different one the user and system config). I can't really test it on my computer since I don't use a display manager, I start x11 using `startx`. Looking at the [nixos module](https://github.com/NixOS/nixpkgs/blob/nixos-22.11/nixos/modules/services/x11/display-managers/default.nix#L91), it seems it executes the extra session command before executing the `.xsession`, so indeed the second execution of `feh` would just replace the wallpaper.

Reply:
Yeah, looks like the Home Manager one will run later. I'm running GNOME so I also can't test.

Reply:
I have a second computer at home, I will try to set it up and test it tonight.